packages/address-consts/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "homepage": "https://github.com/Shopify/quilt/blob/master/packages/address-consts/README.md",
   "files": [
-    "dist/*",
+    "build/*",
```

Comment:
Is there a reason why these are all changing to `build`? Anyway we can keep it as `dist`?

Reply:
Looking through the plugin code I don't believe it's a hard requirement, but I think it's the preferred default. I'll see if keeping it as `dist` changes anything 👍

Reply:
Something to verify. Do to generated root level entry points respect the output folder in the tsconfig. 

* Update the output folder in `packages/web-worker/tsconfig.json` to `dist`
* Run a build 
* Verify that all the root level entry points generated for the following include a `./dist` 

```tsx
  pkg.entry({root: './src/index'});
  pkg.entry({name: 'babel', root: './src/babel-plugin'});
  pkg.entry({name: 'webpack', root: './src/webpack-parts'});
  pkg.entry({name: 'worker', root: './src/worker'});
```

Reply:
Did some digging. Apparently setting `outDir` in `tsconfig.json` only affects TS output. It seems that other outputs are [hardcoded](https://github.com/lemonmade/sewing-kit/blob/main/packages/model/src/base/fs.ts#L46) to go into the `build` folder (the generated entrypoints point to the proper respective folder).

Reply:
Thanks for digging into this. I'm ok with standardizing on our tooling to use `build` as the default output dir. We'll just need to make sure we update all our cases where we hard-coded `dist`. [`@shopify/polyfills`](https://github.com/Shopify/quilt/blob/master/packages/polyfills/src/index.ts#L35) comes to mind. It's hurt us in the [past](https://github.com/Shopify/quilt/commit/33f6eccb500ac576d42719260c31d095fbeabac7). 

We'll probably want an escape hatch out of this in the future though, unless it's already possible today (cc/ @lemonmade)